test/sinon-test.js

```diff
@@ -113,10 +113,5 @@ describe("sinon module", function () {
             });
         });
 
-        describe("useFakeXMLHttpRequest", function () {
-            it("should be the fakeXhr.useFakeXMLHttpRequest export from nise", function () {
```

Comment:
Instread of removing this test, you could change it to verify that the nise implementation is invoked :)

Reply:
Updated.